microsoft/runbook/qemu/qemu.yml

```diff
@@ -0,0 +1,21 @@
+name: qemu default
+include:
+  - path: ../tiers/tier.yml
+variable:
+  - name: keep_environment
+    value: "no"
+  - name: qcow2
+    value: ""
+  - name: extra_user_data
```

Comment:
It looks the `extra_user_data` is a required parameter. It doesn't need to be in this level of runbook. So if someone uses this runbook directly, there is an error message says it's lack of this variable.

Reply:
It isn't actually required. Granted most distros will probably need to use it. But the orchestrator doesn't require this to have a value.